.github/workflows/master_release.yml

```diff
@@ -0,0 +1,72 @@
+name: Master/Release branch workflow
+
+on:
+  push:
+    branches:
+      - master
+      - release/**
+
+env:
+  GIT_USERNAME: ${{ secrets.BOT_GITHUB_USERNAME }}
+  GIT_EMAIL: ${{ secrets.BOT_GITHUB_EMAIL }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  DOCKERHUB_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+  QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
+
+jobs:
+  run_ci:
+    uses: ./.github/workflows/ci.yml
+    secrets: inherit
+  push_to_nexus:
+    name: "Push to Nexus"
+    runs-on: ubuntu-latest
+    needs: [run_ci]
+    if: >
+      always() &&
+      (needs.run_ci.result == 'skipped' || needs.run_ci.result == 'success') &&
+      (! contains(github.event.head_commit.message, '[no release]')) &&
+      (github.ref_name == 'master' || contains(github.ref_name, 'release/')) &&
+      (github.event_name != 'pull_request')
```

Comment:
same for this, since there is only `push` event that can trigger this workflow

Reply:
Removed redundant condition.